libsolidity/interface/CompilerStack.cpp

```diff
@@ -103,6 +107,98 @@ CompilerStack::~CompilerStack()
 	TypeProvider::reset();
 }
 
+void CompilerStack::createAndAssignCallGraphs()
+{
+	for (Source const* source: m_sourceOrder)
+		if (source->ast)
+			for (ASTPointer<ASTNode> const& node: source->ast->nodes())
+				if (auto const* contract = dynamic_cast<ContractDefinition*>(node.get()))
+				{
+					Contract& contractState = m_contracts.at(contract->fullyQualifiedName());
```

Comment:
Can you somehow reduce the indentation level here?

Reply:
You can use `ASTNode::filteredeNodes<ContractDefinitoin>` or something like that